puppet/modules/quickstack/manifests/pacemaker/ceilometer.pp

```diff
@@ -1,8 +1,12 @@
 class quickstack::pacemaker::ceilometer (
   $ceilometer_metering_secret,
-  $memcached_port          = '11211',
-  $db_port                 = '27017',
-  $verbose                 = 'false',
+  $memcached_port                = '11211',
+  $db_port                       = '27017',
+  $verbose                       = 'false',
+  $coordination_backend          = 'redis',
```

Comment:
Super minor nit - could you put these new params in alphabetical order? I think it is easier to find things that way